boards/vcc-gnd/yd_stm32h750vb/board.cmake

```
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(dfu-util "--pid=0483:df11" "--alt=0" "--dfuse")
+board_runner_args(jlink "--device=STM32H735IG" "--speed=4000")
+board_runner_args(openocd --target-handle=_CHIPNAME.cpu0)
+
+include(${ZEPHYR_BASE}/boards/common/dfu-util.board.cmake)
```

Could you test if STM32CubeProgrammer works & add it?
```suggestion
board_runner_args(openocd --target-handle=_CHIPNAME.cpu0)
board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")

include(${ZEPHYR_BASE}/boards/common/dfu-util.board.cmake)
include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
```